content/javascript/concepts/objects/terms/isFrozen/isFrozen.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '.isFrozen()'
+Description: 'Returns a boolean value depending on whether a given object is frozen.'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'Methods'
+  - 'Objects'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **`.isFrozen()`** static method returns `true` if the given object is _frozen_, and `false` if it is not.
```

Comment:
```suggestion
The **`.isFrozen()`** static method returns `true` if the given object is frozen, and `false` if it is not.
```
Could you write when the object is frozen (Conditions), and add the line in the above statement?